common/README.md

```diff
@@ -58,3 +73,42 @@ The following conditions are used:
 
 - **install_crds**: See [install_crds](scripts/install_crds/README.md) for more information
```

Comment:
While not directly part of your changes within the PR: The linked file contains our old module path. Might want to update it now.